packages/terra-demographics-banner/src/_SmallDemographicsBannerDisplay.jsx

```diff
@@ -45,20 +64,47 @@ export default (props) => {
   delete customProps.className;
 
   return (
-    <section className={mainClasses} {...customProps}>
-      <h1 className={cx('person-name')}>
-        <span>
-          { personName }
-          { preferredFirstName && <span className={cx('preferred-first-name')}>{ preferredFirstName }</span> }
-        </span>
-      </h1>
-      <div className={cx('person-details')}>
-        {DemographicsBannerUtils.personDetails(props)}
-        {DemographicsBannerUtils.applicationIdentifiers(props)}
-      </div>
-      <div className={cx('application-content')}>
-        {applicationContent}
+    <section
+      {...customProps}
+      className={mainClasses}
+      tabIndex={-1}
+    >
+      <div className={cx('small-content')}>
+        <div className={cx('column')}>
+          <h1 className={cx('person-name')}>
+            { isConfidential ? <span className={cx('confidential-icon')}><IconFlag /></span> : null }
+            <span>
+              { personName }
+              { preferredFirstName && <span className={cx('preferred-first-name')}>{ preferredFirstName }</span> }
+            </span>
+          </h1>
+        </div>
+        <div className={cx('person-details')}>
+          {DemographicsBannerUtils.personDetails(props)}
+        </div>
+        <div className={cx('identifiers')}>
+          {DemographicsBannerUtils.applicationIdentifiers(props)}
+        </div>
+        <div className={cx('application-content')}>
+          {applicationContent}
+        </div>
       </div>
+      {isSelectable && (
+        <div
+          data-focus-styles-enabled
+          role="button"
+          aria-label={personName || intl.formatMessage({ id: 'Terra.demographicsBanner.info' })}
+          className={cx('info-tile-container')}
+          tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
Does this eslint rule still need to be disabled with `role="button"`?

Reply:
It is no longer needed: https://github.com/cerner/terra-core/pull/2856/commits/5b98d5ff7de9b4397bea7b94cb00934e5930d1e5